Octokit.Tests.Conventions/ModelTests.cs

```diff
@@ -88,6 +83,12 @@ public void AllResponseModelsHavePublicCtorWithAllProperties(Type modelType)
             }
         }
 
+        private bool IsAutoProperty(PropertyInfo prop)
```

Comment:
Stackoverflow copy-paste 😊 Could surely be made more robust if IsAutoProperty is considered useful